docs/contributing.md

```diff
@@ -20,17 +20,18 @@ Please submit bug requests and feature requests through Github as issues. No spe
 
 Please see the information provided in [SECURITY.md](SECURITY.md)
 
-## Development Process for a New Score or Metrics
+## Development Process for a New Score or Metric
 
 A new score or metric should be developed on a separate feature branch, rebased against the main branch. Each merge request should include:
 
  - The implementation of the new metric or score in xarray, ideally with support for pandas and dask
  - 100% unit test coverage
  - A tutorial notebook showcasing the use of that metric or score, ideally based on the standard sample data
- - API documentation (docstrings) using [Napoleon (google)](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) style, making sure to clearly explain the use of the metrics
- - A reference to the paper which described the metrics, added to the API documentation
- - For metrics which do not have a paper reference, an online source or reference should be provided
- - For metrics which are still under development or which have not yet had an academic publication, they will be placed in a holding area within the API until the method has been properly published and peer reviewed (i.e. `scores.emerging`). The 'emerging' area of the API is subject to rapid change, still of sufficient community interest to include, similar to a 'preprint' of a score or metric.
+ - API documentation (docstrings) using [Napoleon (google)](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) style, making sure to clearly explain the use of the metric
+ - A reference to a peer-reviewed paper or other academic reference (e.g. a textbook), added to the API documentation.
+ - For metrics which do not have a paper or academic reference, an online source or reference should be provided
+ - Ideally, references should be open-access. It is reasonable to add an additional open-access link for further reading where relevant.
```

Comment:
Another thought - could consider something along the lines of:
"When providing reference links, use DOIs when available. When DOIs are not available, please use a stable URL if available?